jsk_fetch_robot/jsk_fetch_startup/launch/fetch_bringup.launch

```diff
@@ -408,6 +409,11 @@
     </rosparam>
   </node>
 
+  <include if="$(arg launch_google_analyze_text)"
+           file="$(find ros_google_cloud_language)/launch/analyze_text.launch">
+    <arg name="google_cloud_credentials_json" value="/var/lib/robot/eternal-byte-236613-4bc6962824d1.json" />
```

Comment:
@tkmtnt7000 Can you upload this credential to GoogleDrive and add link to it here?